backend/src/main/java/de/neuefische/cgnjava222/zoomanagement/zoo/employee/EmployeeController.java

```diff
@@ -33,4 +33,10 @@ public ResponseEntity<Void> deleteEmployee(@PathVariable String id) {
         return new ResponseEntity<>(deleteSuccess ? HttpStatus.NO_CONTENT : HttpStatus.NOT_FOUND);
     }
 
+    @PutMapping("{id}")
+    @ResponseStatus(code = HttpStatus.OK)
+    public Employee updateEmployee(@PathVariable String id, @RequestBody Employee employee) {
+        return employeeService.updateEmployee(employee);
```

Comment:
man sollte überprüfen ob pathVar id mit den employee's id in body übereinstimmt

Reply:
Nope, da in unserem Frontend aus dem einen einzigen Employee sowohl Name als auch Id und Position weitergereicht werden. Somit kann die ID nur die gleiche sein.